src/tests/project_euler_tests.zig

```diff
@@ -0,0 +1,26 @@
+const std = @import("std");
+const expectEqualStrings = std.testing.expectEqualStrings;
+
+const dtRunFile = @import("dt_test_utils.zig").dtRunFile;
+
+test "problem_01" {
+    try expectEqualStrings("233168\n", (try dtRunFile("./src/tests/project_euler/problem-01.dt")).stdout);
+}
+
+// Broken by zig impl changes
```

Comment:
Just a note to myself: I should either take a fresh stab at these, or just do it as an unrelated project. Naive solutions here can make the test suite slow

Back when the whole thing was Rust, these were helping me think through things I thought should be in the language but hadn't been implemented yet.